docker/acra-server_transparent.dockerfile

```diff
@@ -0,0 +1,14 @@
+# Create internal synonym for previuosly built image
+ARG DOCKER_REGISTRY_PATH
+ARG VCS_REF
+FROM cossacklabs/acra-server:latest
+
+VOLUME ["/keys"]
+EXPOSE 9090 9393
+
+COPY configs/acra-encryptor.yaml /acra-encryptor.yaml
```

Comment:
where this file used? In `CMD` not used, in `ENTRYPOINT` too. I looked do Acra load config by some default path but it didn't and starts with empty path to config. So by default it will not be used. Do we can load this file in final file which depends on it, in docker-compose by volume definition or something like that?

Reply:
The **docker way** telling us to use layers to add something to container instead of creating more volume entrances for each file that we need in container

Reply:
okay, but it's not docker container for frequent use by many users. it's demo dockerfile which used for several demo docker-compose files. Why I admit it. I looked on file `docker/docker-compose.mysql-nossl-server-ssl-connector_transparent.yml` and tried to see how it configured transparent mode or is it configured. I see next arguments for `acra-server` section:
```
command: >-
...
--encryptor_config_file=/acra-encryptor.yaml
```

And try to find this find in context of docker-compose and didn't find where it declared or generated. And it force me to search other entries for files. And it require to open related dockerfile and read it. Instead find it in context of docker-compose file in `volume` section where we add ssl certificates and all related configs with specific installation (with/out ssl, mysql/postgresql/etc).

@shadinua , what you think about it? How will be better to read new readers such configurations?

Reply:
It looks like both approaches is reasonable for some cases.

But I'd agree with @Lagovas: since we design demo, here we should shift the balance to clarity. Anyway, if there are we have Acra configuration file inside the repository, it would be better to simply mount it to container and explicitly specify in argument list.

Reply:
Ok. Going to change this setup in next commit, tnx for detailed response!